java-squid/src/main/java/org/sonar/java/model/JavaTree.java

```diff
@@ -156,11 +157,11 @@ public boolean isLeaf() {
   public static class CompilationUnitTreeImpl extends JavaTree implements CompilationUnitTree {
     @Nullable
     private final ExpressionTree packageName;
-    private final List<ImportTree> imports;
+    private final List<ImportClauseTree> imports;
     private final List<Tree> types;
     private final List<AnnotationTree> packageAnnotations;
 
-    public CompilationUnitTreeImpl(@Nullable ExpressionTree packageName, List<ImportTree> imports,
+    public CompilationUnitTreeImpl(@Nullable ExpressionTree packageName, List<ImportClauseTree> imports,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/henryju/image-hosting/master/major.png)org/sonar/java/model/JavaTree$CompilationUnitTreeImpl must not use com/sonar/sslr/api/AstNode[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AAstNode_SSLR_class_should_not_be_used)